airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -75,6 +76,30 @@ def build_refresh_request_body(self) -> Mapping[str, Any]:
 
         return payload
 
+    @property
+    def refresh_token_error_status_codes(self) -> Tuple[int]:
+        return (400,)
+
+    @property
+    def refresh_token_error_key(self) -> str:
+        return "error"
+
+    @property
+    def refresh_token_error_values(self) -> Tuple[str]:
+        return ("invalid_grant",)
+
+    def _wrap_refresh_token_exception(self, exception: requests.exceptions.RequestException) -> bool:
+        exception_content = exception.response.json()
```

Comment:
this is not safe, especially when you know the status code most probably is not 200 - body may be empty and `.json()` will raise an error. Please wrap it with try/except